src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20200218140122_AddMissingGeoJsonFlagToDataBlockRequests.Designer.cs

```diff
@@ -0,0 +1,3862 @@
+﻿// <auto-generated />
```

Comment:
This change has already gone in, see 20200214160903_UpdateDataBlockFields.cs
I see this is doing a lot more than just adding the missing geojson flag too.